ui/death_knight/frost/sim.ts

```diff
@@ -86,27 +86,12 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecFrostDeathKnight, {
 		// Default spec-specific settings.
 		specOptions: Presets.DefaultOptions,
 		// Default raid/party buffs settings.
-		raidBuffs: RaidBuffs.create({
-			devotionAura: true,
-			bloodlust: true,
-			markOfTheWild: true,
-			icyTalons: true,
-			leaderOfThePack: true,
-			powerWordFortitude: true,
-			hornOfWinter: true,
-			abominationsMight: true,
-			arcaneTactics: true,
-		}),
+		raidBuffs: RaidBuffs.create({}),
 		partyBuffs: PartyBuffs.create({
 			heroicPresence: false,
 		}),
```

Comment:
`heroicPresence` should be removed from `PartyBuffs`, it's a self-only buff as of Cata.